### DISCUSSION TITLE
ckeditor > config

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17563] [How to]
How to change configuration of ck in yeti 6.5? config.js doesnt work

**Comment by rskrzypczak** [comment]
you can't, YetiForce imposes CKEditor configurations in core files.

**Reply by garryatbrighton** [reply]
![image](https://github.com/user-attachments/assets/f300b161-cbb9-419a-b289-7ae4ff9274dd)
CK in small screen goes too far with toolbars. tswhy I asked. What can be done than?

**Comment by rskrzypczak** [comment]
``` js
App.Fields.Text.Editor.register(container.find('.js-editor')), {
    height: '5em',
    toolbar: 'Min'
});
````

**Reply by garryatbrighton** [reply]
Marius, here is also once confusing thing in ck. When you paste to UI300 table of contents from MSWord, the anchors tags are still clickable causing opening current record as a iframe in the field. More control over what can be set in CK Editor would be beneficial. For me it seems to be potential security issue

**Comment by OkayashiM** [comment]
Maybe you guys can give us method to allow in ckeditor ONLY basic html tags? Like linebrakes, font sizes?

**Reply by mariuszkrzaczkowski** [reply]
it is possible, using classes and data atteybutu in html

**Reply by mariuszkrzaczkowski** [reply]
js-editor js-editor--basic

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/search?q=repo%3AYetiForceCompany%2FYetiForceCRM%20%20js-editor&type=code
